### ISSUE TITLE
Classifier Performance Stats

### ISSUE THREAD

**Issue by a-desmons** [LSSTISSC/Tidalsaurus#8]
This issue lists the performance stats after adding a classifier layer to our SSML model.

Total number of objects in data set: 45,128
Number of objects with known tidal features: 187

- Images were first encoded and the Classifier Layer was trained for 500 epochs
- The Classifier outputs a single number between 0 and 1, with 0 signifying "object with no tidal features", and 1 signifying "object with tidal features"

From the 187 images with known tidal features, **12** were set aside for testing performance and testing for overfitting - These are shown [here](https://user-images.githubusercontent.com/61219404/161187595-185e3625-68f2-474b-8e56-1f2734b7b82a.png)

All images with known tidal features (i.e. the 187) were found within the first **7038** images with a ranking closest to 1.

The table below shows:
- Test image number: The image used for testing
- Similarity index: Its ranking determined by the classifier, with 0 being the "best" and 45,128 being the "worst"

| Test image number  | Similarity Index | 
| -------------------- | ------------- |
| 1 | 3982 |
| 8 | 307 |
| 18 | 4030 |
| 27 | 3800 |
| 76 | 451 |
| 86 | 997 |
| 97 | 418 | 
| 117 | 21 |
| 119 | 7037 |
| 161 | 869 |
| 163 | 113 |
| 177 | 391 |

And a histogram of the classifier's performance in finding the images with known tidal features:

![71788D37-204A-4764-A1B8-E2A529CDBBFD_4_5005_c](https://user-images.githubusercontent.com/61219404/161189666-1b58ab79-4dec-4bee-87ea-e855e2725c57.jpeg)

**Comment by a-desmons** [comment]
Comparing the performance of just the model (with single galaxy as query) to model+classifier.

x-axis: Similarity index in thousands (i.e. 1 = 1000)
y-axis: Fraction of galaxies with tidal features found

![download-1](https://user-images.githubusercontent.com/61219404/161666465-498c79e0-a0f4-442e-85a2-3058d7e4c148.png)

Zooming in on first half of the plot:
![download-2](https://user-images.githubusercontent.com/61219404/161667398-db21e970-1ad9-4742-ac74-d182c49ec74c.png)

**Comment by a-desmons** [comment]
Sampling results by taking 6 images every 10,000 similarity-ranked images

![sample_by_10000](https://user-images.githubusercontent.com/61219404/161681150-dc356471-ea90-422d-adaf-4aadd6c3edab.png)
